cpp/src/c_api/graph_sg.cpp

```diff
@@ -102,13 +102,19 @@ struct create_graph_functor : public cugraph::c_api::abstract_functor {
       rmm::device_uvector<vertex_t>* number_map =
         new rmm::device_uvector<vertex_t>(0, handle_.get_stream());
 
-      std::tie(*graph, new_number_map) = cugraph::
-        create_graph_from_edgelist<vertex_t, edge_t, weight_t, store_transposed, multi_gpu>(
+      std::tie(*graph, std::ignore, new_number_map) =
+        cugraph::create_graph_from_edgelist<vertex_t,
+                                            edge_t,
+                                            weight_t,
+                                            int32_t,
```

Comment:
This hard-codes int32_t, which is fine for now (as you've discussed with @alexbarghi-nv.  Ultimately we should probably allow this to be arbitrary, but I think we can defer that to later.

Reply:
What about doing `using edge_type_t = int32_t` then passing `edge_type_t`? I guess this will make the intention clearer.

Reply:
I think that would be a bit better.  That would make the references easier to isolate when we ultimately add support for variable types.